submissions/krylenger/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,1342 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body class="body">
```

Comment:
Avoid using `<body>` as container.
```suggestion
  <div class="container">
      <div class="left-col"></div>
      <div class="right-col"></div>
  </div>
```

Reply:
Thanks! 
Is it fine to do like this: 
`<div class="container">
      <section class="left-col"></section>
      <main class="right-col"></main>
  </div>`
?

or better stick to divs in this case?

Reply:
Yes it's fine, I mean not necessary to use long class names for simple elements) and use div as wrapper instead of `body`.

Reply:
✅ Done